src/styles/styles.scss

```diff
@@ -5,7 +5,7 @@
 
 :root {
   body {
-    // font weight
+    font-family: var(--kui-font-family-text, $kui-font-family-text);
```

Comment:
issue: we don't want to define a font family on the body; this is controlled by the host app